### ISSUE TITLE
build "ios.release" error

### ISSUE THREAD

**Issue by zhenjunMa** [berty/berty#3131]
### GO version
go version go1.16 darwin/amd64

### path
./js

### problem
I want to install berty on my iphone, so I execute `make ios.release`
```
cd .. && go mod download
cd .. && go run golang.org/x/mobile/cmd/gomobile init
# Download required go-libtor deps if needed
if [ "v1.0.345" != "" ]; then \
		set -o pipefail; \
		mkdir -p ios/tor-deps /var/folders/06/71j__qmj7ws44bc9_hmd2d480000gp/T//tor-deps/ios && \
		libs=$(wget -O - "https://github.com/berty/go-libtor/releases/tag/v1.0.345" 2> /dev/null | grep -o 'ios-.*-universal\.tar\.gz' | uniq) && \
		for lib in $libs; do \
			wget "https://github.com/berty/go-libtor/releases/download/v1.0.345/$lib" -O "/var/folders/06/71j__qmj7ws44bc9_hmd2d480000gp/T//tor-deps/ios/$lib" && \
			tar xvf "/var/folders/06/71j__qmj7ws44bc9_hmd2d480000gp/T//tor-deps/ios/$lib" -C ios/tor-deps || exit 1; \
		done && \
		echo v1.0.345 > ios/tor-deps/version; \
	fi
make: *** [ios/Frameworks/Bertybridge.framework] Error 1
```

anyone know how to solve this?

**Comment by D4ryl00** [comment]
Sorry for the delay!
Do you have `wget` on your system? If no, you can install it with this command: `brew install wget`

**Comment by zhenjunMa** [comment]
yeah, it works, thank you!

**Comment by aeddi** [comment]
Also fixed by #3190 that make the download script compatible with both `wget` and `curl` and displays a clear error message if neither is installed.
